src/cplscheme/config/CouplingSchemeConfiguration.cpp

```diff
@@ -591,8 +598,10 @@ void CouplingSchemeConfiguration::addTagMaxIterations(
     xml::XMLTag &tag)
 {
   using namespace xml;
-  XMLTag            tagMaxIterations(*this, TAG_MAX_ITERATIONS, XMLTag::OCCUR_NOT_OR_ONCE);
+  XMLTag tagMaxIterations(*this, TAG_MAX_ITERATIONS, XMLTag::OCCUR_NOT_OR_ONCE);
+  tagMaxIterations.setDocumentation("Allows to specify a maximum amount of iterations. The simulation will end if this limit is exceeded.");
```

Comment:
Is this not maximum iterations until the time-window is incremented, regardless if convergence is reached or not?

Reply:
@BenjaminRueth can you enlighten us?

Reply:
It's the iterations per time window. If reached we go to the next time window.